rex/sam_resource.py

```diff
@@ -555,9 +603,12 @@ def _get_var_ts(self, var, *var_slice):
                 raise ResourceKeyError(msg)
 
             sites = np.array(self.sites)
+            if len(var_slice) == 2:
```

Comment:
Did something break here? Why isn't there a condition for a 1D slice?

Reply:
I have no idea how this didn't break earlier. It broke for me when I tried to do sam_res[var], in that case, var_slice = (slice(None),) and it tried to do sites[var_slice[1]].

My understanding is that since these are time-series they are always 2D, so a 1D slice would  just slice in time, and a 2D slice would slice time and space

Reply:
Gotcha. Slice dimensions are tricky especially with the getters... Sounds good!